gems/aws-sdk-s3/lib/aws-sdk-s3/plugins/http_200_errors.rb

```diff
@@ -27,12 +27,25 @@ def check_for_error(context)
               error_code = xml.match(/<Code>(.+?)<\/Code>/)[1]
               error_message = xml.match(/<Message>(.+?)<\/Message>/)[1]
               S3::Errors.error_class(error_code).new(context, error_message)
-            elsif xml.strip == ''
-              Seahorse::Client::NetworkingError.new(
-                S3::Errors
+            elsif incomplete_response?(xml, context)
+              S3::Errors
                   .error_class('InternalError')
-                  .new(context, 'Empty response body')
-              )
+                  .new(context, 'Incomplete or empty response body')
```

Comment:
Will this be retryable? It looks to be.

Reply:
Yeah - its retryable.  The spec checks that gets retried.